### DISCUSSION TITLE
Welcome to TeamTalk Discussions!

### DISCUSSION THREAD

**Discussion by bear101** [BearWare/TeamTalk5#1009] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by BTIofficial** [comment]
Good morning, my name is Rangga from Jakarta Indonesia, I want to give advice and input for TeamTalk for android?

So there's a few things I'm going to talk about, related to feature additions and apical bug fixes.

1. We want the broadcast message feature on PC to be accessible on Android too.
2. We can use the video streaming feature on the desktop on Android.
3. View broadcast messages in channel messages.
4. Admin features on each server to be further equipped, such as creating usernames, intercepting, editing server properties, and being able to enter operator passwords.
5. Added a free music bot (YouTube) feature, to be accessed by all TeamTalk PC and Android application users.

Maybe that's all I can say, thank you in advance, hopefully it can be useful for all users and developers. More or less sorry and thank you. Good morning!

**Reply by bear101** [reply]
If you filter [Issues](https://github.com/BearWare/TeamTalk5/issues?q=is%3Aissue) by _Enhancement_ and _TeamTalkAndroid_ then you'll find these issues in the list.

**Comment by SalehAcehUtara** [comment]
hahahahaha.

Bro,  @BTIofficial  Your suggestion is very good and constructive.

However, the proposed number:
5. Added a free music bot (YouTube) feature, to be accessed by all TeamTalk PC and Android application users.
Very unreasonable.

Audio Player/Bot circulating in TeamTalk is not entirely the responsibility of the TeamTalk developer itself.
TeamTalk developers only focus on developing TeamTalk Applications themselves, And Bot developers only try to develop for their own bots.
The bot/Audio Player circulating on TeamTalk must also use a wired audio line.
Is it possible on Android there is a Virtual Audio Cable?

Humans are given reason to think, use it as well as possible.
Sorry, only answering what shouldn't be asked!

**Reply by DalisonCPU** [reply]
I submitted several suggestions in a post here, and the developer did not respond. Look there, and maybe you can say that it doesn't make sense to be created either. I think it's easier to close the suggestion system than to criticize the suggestions we are making to improve the product.

**Comment by lbk2907** [comment]
Hey there! Good morning here from Malaysia. I have a suggestion here. Make the show variables check box in the server properties is check by default. I have saw that the MOTD variables was accidentally removed by some of the TeamTalk administrators or user that have an ability to update server properties because of they didn't realise this check box is there. They just simply update the server properties especially edit the MOTD and click on OK button. When the dialog pops up that said "Change message of the day?", they will just directly hit the ok button. Thanks.
